import java.io.IOException;

class cadastro{
    public static void main(String args[]) throws IOException{
        System.out.println("Os dados foram enviados com sucesso!");
    }
}
